bindings/swift/LDKNodeFFI.xcframework/Info.plist

```diff
@@ -8,7 +8,7 @@
 			<key>LibraryIdentifier</key>
```

Comment:
q: what are we using the checked in `.xcframework` for?

Reply:
> Mh, we're not checking in the full framework, just the necessary umbrella headers. Correct me if I'm wrong, but my understanding is this is necessary so that it can be used as a local dependency, without SwiftPM? (cc @reez)

Yeah this is similar to how BDK swift ffi is set up related to how they allow for local import of a Swift Package

Reply:
👍 that makes sense